go.mod

```diff
@@ -9,3 +9,5 @@ require (
 	gopkg.in/ns1/ns1-go.v2 v2.0.0-20191126161805-25b9eac84517
 	gopkg.in/yaml.v2 v2.2.7 // indirect
 )
+
+go 1.13
```

Comment:
I think we want to keep targeting/devving with 1.12, but might be worth discussing further ...

Reply:
Okay, I don't think it makes a big difference either way, but I'll change it to 1.12